java/.idea/jarRepositories.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The addition of this file appears orthogonal to the message in the commit - feel free to correct me if I'm wrong. For future reference, you should split the addition of this file out into a separate commit. If you want to practice your git skills, you can remove this file from this commit and then add it to its own commit. I've not done it this way before, but this looks interesting: https://devconnected.com/how-to-remove-files-from-git-commit/

I dunno:
![image](https://user-images.githubusercontent.com/45576380/84687272-73c4cd80-af0b-11ea-826f-6154ce83bec1.png)